### ISSUE TITLE
Sub-GHz Incorrect range for IE (Ireland)

### ISSUE THREAD

**Issue by OzmoOzmo** [flipperdevices/flipperzero-firmware#2312]
### Describe the bug.

Cannot transmit to 433 Mhz in Ireland using Sub-Ghz app.
Attempting to TX to 433.9Mhz
433-434 is permitted in EU (Ireland)
Flipper incorrectly says frequency is blocked for our region
Even though SubGhz Region menu lists this frequency as within legal ranges.

original firmware-Installed (0.75 ofw) as normal (no VPN used on installation pc)
See screenshots for details... It lists correct IE and the correct permitted ranges - but still will not TX on 433Mhz.

Lock Device we testing with is a unfitted model WIR222 receiver and 2 button key fob.
Thanks.
![Ireland_transmission_tables](https://user-images.githubusercontent.com/6533854/212783835-a55f5451-325f-4d50-b915-57903f58d7e2.png)
![PXL_20230116_234922238](https://user-images.githubusercontent.com/6533854/212783838-c0ed47ca-8f84-4dfd-87bd-9b895041f9b8.jpg)
![PXL_20230116_234948812](https://user-images.githubusercontent.com/6533854/212783842-48e4f4e7-89fb-491d-9617-f9de9595fd27.jpg)
![PXL_20230116_235003500](https://user-images.githubusercontent.com/6533854/212783843-4e24b608-ba06-45c4-a54c-cd49496e2ada.jpg)
![PXL_20230116_235559324](https://user-images.githubusercontent.com/6533854/212783844-7280c29c-1746-41d2-822e-fa45c529ccf4.jpg)

### Reproduction

1. Start SubGhz Read - Capture Key Fob
2. Replay Key Fob (with or without saving first)
3. Flipper says Blocked and will not TX.

### Target

A 2 Channel Receiver - identified as a "model WIR222" receiver and 2 button key fob.

### Logs

```Text
.sub file is as follows:

Filetype: Flipper SubGhz Key File
Version: 1
Frequency: 433920000
Preset: FuriHalSubGhzPresetOok650Async
Protocol: CAME
Bit: 40
Key: 00 00 00 53 0A FA AE EA
```

**Comment by OzmoOzmo** [comment]
Some more info - hope it helps - log (from putty) - shows the following error
**[E][SubGhzProtocolCAME] Wrong number of bits in key**

From source code - it seems Flipper, when recording, is thinking this device is using "CAME" protocol - and when it sends - it realizes the bit count is not CAME compatible and then proceeds to give _the wrong error message_.

So issue is two fold - 
1. incorrectly detecting the protocol (should be RAW?)
2. showing error "Restricted in Region" when it should be "Cannot send due to bad Protocol/Corrupt save" or similar.

Reading and Replying as RAW worked! So my device has no rolling codes etc.

**Comment by Skorpionm** [comment]
the protocol is not Came.  even if it looks a lot like him.  I did not specifically restrict the registration of such "similar" protocols, but prohibited the transfer.  post a photo of the remote control, receiver, what names are there, a photo of the board and, most importantly, RAW recordings of parcels.  5+ pieces of holding each of the remote control buttons for 3 seconds, and it would be nice if there were records of different remote controls

**Comment by skotopes** [comment]
Yep, incorrect message that we showing to you. Issue with the key file. We'll update message.

**Comment by skotopes** [comment]
Issue was fixed in #2443 please check and let us know if it still persist
